assets/js/editor-components/utils/index.js

```diff
@@ -217,3 +219,29 @@ export const formatTitle = ( page, pages ) => {
 		pages.filter( ( p ) => p.title.raw === page.title.raw ).length === 1;
 	return page.title.raw + ( ! isUnique ? ` - ${ page.slug }` : '' );
 };
+
+/**
+ * Gets whether the block (passed by clientId) is the Cart, Checkout, or any of their inner blocks.
+ *
+ * @param {string} clientId The clientId of the block.
+ * @return { { isCheckout: boolean, isCart: boolean } } An object with isCheckout and isCart booleans.
+ */
+export const useIsCartOrCheckoutOrInnerBlock = ( clientId ) => {
```

Comment:
Maybe we need a better name for this?

Reply:
Now that this is used in a single place, do we need to have a hook?

Reply:
Good point, I thought I used it in the `withBlockSettings` hoc but I didn't. Removed it now.